config/settings.yml

```diff
@@ -19,62 +19,62 @@
       master:
     manageiq-v2v:
       master:
-  :productization_name: manageiq
+  :productization_name: xaasio-cmp
   :remotes:
-    :upstream: git@github.com:ManageIQ
+    :upstream: https://gitlab.xaasio.com/xaasio-cmp
```

Comment:
These values should not be changed.  If you need to override anything from this file you can create your own override settings.xassio-cmp.yml file, which I thikn will get automatically pulled in and override anything in the base settings.yml.